root/usr/share/tancredi/src/Entity/NethVoiceAuth.php

```diff
@@ -60,8 +60,8 @@ public function __invoke($request, $response, $next)
             }
         } elseif ($request->hasHeader('Secretkey') && $request->hasHeader('User')) {
             $dbh = new \PDO(
-                'mysql:dbname=asterisk;host=localhost',
-                'tancredi',
+                'mysql:dbname=asterisk;host='.isset($this->config['auth_nethvoice_dbhost'] ? $this->config['auth_nethvoice_dbhost'] : 'localhost',
+                isset($this->config['auth_nethvoice_dbuser'] ? $this->config['auth_nethvoice_dbuser'] : 'tancredi',
```

Comment:
https://www.php.net/manual/en/language.operators.comparison.php#language.operators.comparison.coalesce

if 5.6 PHP version supports it...

```suggestion
                'mysql:dbname=asterisk;host=' . ($this->config['auth_nethvoice_dbhost'] ?? 'localhost'),
                $this->config['auth_nethvoice_dbuser'] ?? 'tancredi',
```

Reply:
We now use 7.33 for tancredi and it is supported